app/assets/stylesheets/application.css.sass

```diff
@@ -9,8 +9,17 @@
   font-size: 75%
   font-weight: bold
 
-// Fixes the background scrolling when opening a modal 
+// Fixes the background scrolling when opening a modal
 body.modal-open
   overflow: hidden
   position: fixed
-  width: 100%
\ No newline at end of file
+  width: 100%
+
+// Add spacing around nav tabs in panel
+.panel
+  >.nav.nav-tabs
+    padding-top: 20px
+    margin-bottom: 10px
```

Comment:
It seemed to look better when separated. But you can revert this change if you want.